# 消息头
std_msgs/Header header

# 图像序列号
uint64 image_seq

# 检测目标列表
PerceptionTarget[] targets
